rust/src/shards/gui/layouts/table.rs

```diff
@@ -404,18 +428,17 @@ impl Shard for Table {
               }
             }
           })
-        } else {
-          // fallback to default columns with no header
-          for _ in 0..seq.len() {
-            builder = builder.column(Column::remainder());
-          }
-          builder.header(0.0, |_| {})
```

Comment:
It is still necessary to do something if `:Columns` isn't set. Otherwise it will have phantom headers that take some vertical space.

Could be simplified though with a condition around the header creation above:
```rust
let row_height = if columns.len() > 0 { 20.0 } else { 0.0 };
builder.header(row_height, |mut header_row| {
// ...
}
```